conda/meta.yaml

```diff
@@ -28,6 +28,7 @@ requirements:
     - cmake
     - cppcheck [linux64]
     - conan>=1.45.0
+    - ccache
```

Comment:
This is a bit odd, since ccache is *not* used/required for `conda build`. Add this in `developer-extra.yml` instead?